projects/ks89/angular-modal-gallery/src/lib/components/previews/previews.component.ts

```diff
@@ -194,6 +203,9 @@ export class PreviewsComponent extends AccessibleComponent implements OnInit, On
    * @param Action action that triggered the source event or `Action.NORMAL` if not specified
    */
   onImageEvent(preview: InternalLibImage, event: KeyboardEvent | MouseEvent, action: Action = Action.NORMAL): void {
+    // It's suggested to stop propagation of the event, so the
```

Comment:
Why is this necessary? I implemented all other features without this.
Did you find a specific reference about this somewhere?

Reply:
I believe it's necessary because the CDK overlay below is awaiting for any click to close the whole gallery, and receives it when the user clicks on the description text for instance (at least on Windows Chrome/Firefox). I applied it on the whole template-surrounding div here, because most likely a click on that zone is expected to lead to the connected preview and not to close the gallery unintentionally.